style.css

```diff
@@ -389,3 +425,245 @@ textarea {
   display: flex;
   justify-content: center;
 }
+
+/* ---------- desktop view----------------- */
+
+@media (min-width: 768px) {
+  .container {
+    width: 100%;
+    margin: auto;
+  }
+
+  nav {
+    padding: 0;
+    margin: 0;
+  }
+
+  header nav .toolbar {
+    display: flex;
+    padding: 0;
+    align-items: center;
+    height: 100%;
+    width: 80%;
+    margin: auto;
+  }
+
+  header nav .toolbar ul {
+    display: flex;
+    height: 100%;
+    justify-content: right;
+    margin-top: 0;
+    margin-bottom: 0;
+    flex: 1;
+  }
+
+  .con-links li {
+    margin: 0% 10px;
+  }
+
+  header nav .toolbar ul li {
+    display: flex;
+    align-items: center;
+    padding: 8px 34px;
+  }
+
+  header nav .toolbar ul li a {
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .menu-bar img {
+    display: none;
+  }
+
+  .headline {
+    display: flex;
+    padding-bottom: 150px;
+    background-position: 100% 80%;
+    background-size: 558px;
+  }
+
+  .headline-body {
+    width: 556px;
+    padding-right: 400px;
+    order: 99;
+  }
+
+  .headline-body h1 {
+    width: 60%;
+    margin-bottom: 0;
+    margin-top: 0;
+  }
+
+  .headline-body p,
+  .headline h3 {
+    width: 75%;
+  }
+
+  .headline-body p {
+    padding-top: 0;
+  }
+
+  .headlink {
+    margin: 0 24px;
+  }
+
+  .headlink .social-links {
+    display: block;
+  }
+
+  .work-container {
+    padding-bottom: 128px;
+    grid-template-columns: auto auto auto;
+    gap: 3% 3%;
+  }
+
+  .item {
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .work-item-1 {
+    grid-column: 1/4;
+    padding-top: 12px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .no-btn button {
+    display: none;
+  }
+
+  .item:nth-child(2) {
+    background-image: url(image/Img_Placeholder_2.svg);
+    display: flex;
+    position: relative;
+    height: 450px;
+  }
+
+  .work-item-2 .project-btn {
+    position: absolute;
+    bottom: 0;
+  }
+
+  .item:nth-child(3) {
+    background-image: url(image/Img_Placeholder_3.svg);
+  }
+
+  .item:nth-child(4) {
+    background-image: url(image/Img_Placeholder_4.svg);
+  }
+
+  .item:nth-child(5) {
+    background-image: url(image/Img_Placeholder_5.svg);
+  }
+
+  .item:nth-child(6) {
+    background-image: url(image/Img_Placeholder_6.svg);
+  }
+
+  .item:nth-child(7) {
+    background-image: url(image/Img_Placeholder_7.svg);
+  }
```

Comment:
- [ ] Please, kindly add the hover effect on your cards to show the `see project` button so that if a user clicks on the button, it takes them to the link of your portfolio GitHub repository.